src/api/spec/controllers/webui/package_controller_spec.rb

```diff
@@ -518,4 +520,32 @@ def remove_file_post
       end
     end
   end
+
+  describe 'GET #trigger_services' do
+    let(:post_url) { "http://localhost:3200/source/#{source_project}/#{service_package}?cmd=runservice&user=#{user}" }
```

Comment:
@mdeniz is right, we should change the other places, too! 
Could you try if it works with a RegEx?